mypy/git.py

```diff
@@ -0,0 +1,119 @@
+"""Utilities for verifying git integrity."""
+
+# Used also from setup.py, so don't pull in anything additional here (like mypy or typing):
+import os
+import pipes
+import subprocess
+import sys
+
+
+def is_git_repo(dir: str) -> bool:
+    """Is the passed directory version controlled with git?"""
```

Comment:
Sounds like someone..."expelled" a directory or something. :) What about "the given directory"?